service/src/main/kotlin/fi/espoo/evaka/koski/KoskiUpdateService.kt

```diff
@@ -29,7 +29,10 @@ class KoskiUpdateService(
         asyncJobRunner.scheduleKoskiUploads = { db, msg -> scheduleKoskiUploads(db, msg.params) }
     }
 
-    fun scheduleKoskiUploads(db: Database, params: KoskiSearchParams) {
+    fun scheduleKoskiUploads(db: Database, params: KoskiSearchParams) = db.connect {
+        scheduleKoskiUploads(it, params)
+    }
```

Comment:
It's probably just too hot in here and I'm being stupid but why doesn't this care about "fi.espoo.integration.koski.enabled"?

Reply:
The check is done in the inner `scheduleKoskiUploads(it, params)` call

Reply:
:facepalm: